jobs/deploy-fastly.groovy

```diff
@@ -45,26 +45,34 @@ phabricator/diff/&lt;id&gt; (using the latest ID from the diff's "history" tab o
     ""
 
 ).addChoiceParam(
-    "TARGET",
+    "SERVICE",
     """\
 <ul>
-  <li> <b>test</b>: https://www-test.khanacademy.org, www-boxes.khanacademy.org
-  <li> <b>prod</b>: https://www.khanacademy.org
-  <li> <b>staging</b>: https://www-staging.khanacademy.org [compute@edge only]
+  <li> <b>khanacademy.org (vcl)</b>
+  <li> <b>khanacademy.org (compute)</b>
+  <li> <b>blog</b>
+  <li> <b>content-property</b>
+  <li> <b>international</b>
+  <li> <b>kasandbox</b>
+  <li> <b>kastatic</b>
+  <li> <b>khan.co</b>
+  <li> <b>sendgrid</b>
 </ul>
 """,
-    ["test", "prod", "staging"]
+    ["khanacademy-org-vcl", "khanacdemy-org-compute",
+     "blog", "content-property", "international", "kasandbox", "kastatic",
+     "khan-co", "sendgrid"],
 
 ).addChoiceParam(
-    "SERVICE",
+    "TARGET",
     """\
 <ul>
-  <li> <b>fastly-vcl</b>
-  <li> <b>fastly-compute</b>
-  <li> <b>kastatic</b>
+  <li> <b>test</b>: khanacademy-org-vcl and khanacademy-org-compute <b>only</b>
+  <li> <b>staging</b>: khanacademy-org-compute <b>only</b>
+  <li> <b>prod</b>: all services
```

Comment:
i have always wondered if this list could be dynamically created because on another input like the `SERVICE` field.

Reply:
No, sadly.